Patches/Geth/Ammo_Geth_CE.xml

```diff
@@ -0,0 +1,414 @@
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>Geth</li>
+    </mods>
+		<match Class="PatchOperationSequence">
+		<operations>		
+
+		<li Class="PatchOperationAdd">
+		<xpath>Defs</xpath>
+		<value>
+
+  <ThingCategoryDef>
+    <defName>AmmoML77CE</defName>
+    <label>ML-77 Missile</label>
+    <parent>AmmoRockets</parent>
+    <iconPath>UI/Icons/ThingCategories/CaliberRocket</iconPath>
+  </ThingCategoryDef>
+
+	<!-- ==================== AmmoSet ========================== -->
+
+	<CombatExtended.AmmoSetDef>
+		<defName>AmmoSet_GethRifle</defName>
+		<label>Plasma Rifle Power Cell</label>
```

Comment:
Because Mass Effect weapons (and geth weapons specifically) have universal ammo save for the rocket launcher, this ammo should probably be revised to be universal, with different weapons using different amounts of "capacity" when they fire. It's not really a perfectly solution in the absence of the eventual goal of a power cell system, but it's closer to the mark.

Are you familiar with using `<ammoConsumedPerShotCount>` for weapons?

Reply:
They do have universal ammo, that’s exactly what I did?